ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/button/v1/button/button.html

```diff
@@ -14,21 +14,32 @@
   ~ limitations under the License.
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 <sly data-sly-use.button="com.adobe.cq.forms.core.components.models.form.Button"
-  data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
-  data-sly-use.iconTemplate="icon.html"
-  data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"></sly>
-<button 
-  type="button"
-  value="${button.value}"
-  id="${component.id}"
-  class="cmp-button"
-  disabled="${!button.enabled}"
-  title="${button.tooltipVisible ? '' : button.tooltipText}"
-  data-cmp-visible="${button.visible}"
-  data-cmp-enabled="${button.enabled}"
-  data-cmp-is="adaptiveFormButton"
-  aria-label="${button.label.value}"
-  data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
-<sly data-sly-call="${iconTemplate.icon @ icon=button.icon}"></sly>
-<span data-sly-test="${button.label.value}" class="cmp-button__text">${button.label.value}</span>
-</button>
\ No newline at end of file
+     data-sly-use.renderer="${'button.js'}"
+     data-sly-use.label="${renderer.labelPath}"
+     data-sly-use.shortDescription="${renderer.shortDescriptionPath}"
+     data-sly-use.longDescription="${renderer.longDescriptionPath}"
+     data-sly-use.questionMark="${renderer.questionMarkPath}"
+     data-sly-use.component="com.adobe.cq.wcm.core.components.models.Component"
+     data-sly-use.iconTemplate="icon.html"
+     data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"></sly>
+<div class="cmp-button"
```

Comment:
cmp-button was borrowed from wcm button where it was applicable to actual button. but now, it is being applied to a div.  I think we should introduce our own bem for button also. `cmp-adaptiveform-button` since we are no longer following wcm structure.

Reply:
yes, that would also align with the other form component's BEM